libnmstate/netapplier.py

```diff
@@ -329,6 +346,8 @@ def assert_ifaces_state(ifaces_desired_state, ifaces_current_state):
             iface_dstate, iface_cstate)
         iface_dstate, iface_cstate = _remove_iface_ipv6_link_local_addr(
             iface_dstate, iface_cstate)
+        iface_cstate = sanitize_dhcp_state(
+            {iface_cstate['name']: iface_cstate})[iface_cstate['name']]
```

Comment:
Can we use `ifname` instead of `iface_cstate['name']`? Not a must, it is just less noisy.